pkg/util/pod.go

```diff
@@ -47,10 +47,10 @@ func GetPodKubeRelativePath(pod *corev1.Pod) string {
 }
 
 // @podParentDir kubepods-burstable.slice/kubepods-pod7712555c_ce62_454a_9e18_9ff0217b8941.slice/
-// @output /sys/fs/cgroup/cpuacct/kubepods.slice/kubepods-burstable.slice/kubepods-pod7712555c_ce62_454a_9e18_9ff0217b8941.slice/cpuacct.proc_stat
-func GetPodCgroupCPUAcctProcStatPath(podParentDir string) string {
```

Comment:
This function was used in an unit test, you also need to edit it.
https://github.com/koordinator-sh/koordinator/blob/5d624d7161974507dabda266c1cb8aef711a7de7/pkg/util/pod_test.go#L286-L293

Reply:
OK, I see, my main branch is not sync with github main.

Reply:
fixed